netbox/netbox/settings.py

```diff
@@ -112,9 +112,9 @@
 DEVELOPER = getattr(configuration, 'DEVELOPER', False)
 DOCS_ROOT = getattr(configuration, 'DOCS_ROOT', os.path.join(os.path.dirname(BASE_DIR), 'docs'))
 EMAIL = getattr(configuration, 'EMAIL', {})
-EVENTS_PIPELINE = getattr(configuration, 'EVENTS_PIPELINE', (
+EVENTS_PIPELINE = getattr(configuration, 'EVENTS_PIPELINE', [
     'extras.events.process_event_queue',
-))
+])
```

Comment:
I think we want to ensure that `process_event_queue()` is always included in the pipeline; otherwise, event rules would break. The config parameter should be treated as a list of _additional_ items to include. (I realize this deviates from how it's currently handled.)